iphone/Maps/Classes/CustomViews/CircularProgress/MWMCircularProgressView.mm

```diff
@@ -9,27 +9,23 @@
 static CGFloat const kLineWidth = 2.0;
 static NSString * const kAnimationKey = @"CircleAnimation";
 
-static inline CGFloat angleWithProgress(CGFloat progress)
-{
-  return 2.0 * M_PI * progress - M_PI_2;
-}
-
+static inline CGFloat angleWithProgress(CGFloat progress) { return 2.0 * M_PI * progress - M_PI_2; }
```

Comment:
Do you really need statics here? Is it possible to use anonymous namespaces instead?